NorsokSCDLibrary.aml

```diff
@@ -28135,4 +28293,8 @@
       </SystemUnitClass>
     </SystemUnitClass>
   </SystemUnitClassLib>
+  <AttributeTypeLib Name="AttributeTypeLib">
```

Comment:
Should this library have "Norsok" in the name?
Are we sticking to a convention for library names?

Reply:
All existing upper level libraries don't contain "Norsok" in the names (e.g. DocumentClassLibrary, ReferenceClassLibrary). Adding "Norsok" to AttributeTypeLib seems not necessary and inconsistent.